content/en/opentelemetry/collector_exporter/configuration.md

```diff
@@ -9,13 +9,238 @@ further_reading:
   text: "OpenTelemetry Collector Configuration"
 ---
 
-## OpenTelemetry Collector configuration
+The OpenTelemetry Collector receives, processes, and exports telemetry data from your applications. To send this data to Datadog, you need to configure several components within the Collector:
 
-To learn more about configuring the Collector, read the [OpenTelemetry Collector Configuration][3] documentation.
+- [Datadog Exporter][1]: Forwards trace, metric, and logs data from OpenTelemetry SDKs on to Datadog (without the Datadog Agent).
+- [Datadog Connector][29]: Calculates Trace Metrics from collected span data.
+
+## Setting up the OpenTelemetry Collector
+
+To run the OpenTelemetry Collector with the Datadog Exporter and Datadog Connector:
+
+### Step 1 - Download the OpenTelemetry Collector
+
+Download the latest release of the OpenTelemetry Collector Contrib distribution, from [the project's repository][3].
+
+### Step 2 - Configure the Datadog Exporter and Connector
+
+To use the Datadog Exporter, add it to your [OpenTelemetry Collector configuration][4]. Create a configuration file and name it `collector.yaml`. Use the example file which provides a basic configuration that is ready to use after you set your Datadog API key as the `DD_API_KEY` environment variable:
+
+{{< code-block lang="yaml" filename="collector.yaml" collapsible="true" >}}
+receivers:
+  otlp:
+    protocols:
+      http:
+        endpoint: 0.0.0.0:4318
```

Comment:
Do we want to include a warning like the one you can see here https://opentelemetry.io/docs/collector/configuration/#basics in our docs ?

Reply:
Ah, great point. I forgot about that inclusion from your PR. I'll add something to this effect!

Reply:
I added an alert here: [layouts/shortcodes/otel-endpoint-note.md](https://github.com/DataDog/documentation/pull/24369/files#diff-301b14f970dc8fc3a994849c8c61caa34d0a87cd84efe0057c0559275df34581). Let me know what you think! I left out the part about changing the default in the future to keep our docs focused on the current state.